### ISSUE TITLE
[Android] App crash with error `ILLegal callback invocation from native module.`

### ISSUE THREAD

**Issue by kevin-mcs** [opentok/opentok-react-native#639]
# Bug Report

**Current behavior**
> Android app crashes whenever I try to Connect for video session in my application it crashes in sometimes.

**Steps to reproduce**
>  My Project version and package version
    `"opentok-react-native": "0.20.4",`
    `"react": "17.0.2"`
    `"react-native": "0.68.2",`

**What is the current *bug* behavior?**
> Android app crashes whenever I try to Connect for video session in my application it crashes after some seconds with below error in Android studio log-cat.

```
FATAL EXCEPTION: mqt_native_modules
Process: com.fiverr.doctorapp, PID: 29657
java.lang.RuntimeException: ILLegal callback invocation from native module. This callback type only permits a single invocation from native code
    at com.facebook.react.bridge.CallbackImpl. invoke (CallbackImpl. java:26)
    at com.opentokreactnative.OTSessionManager.onConnected(OTSessionManager.java:593)
    at com.opentok.android.Session.c (Unknown Source:4)
    at com.opentok.android.Session.$r8$lambda$S(g80MmHLagUV9v3S490eoPGMuc(Unknown Source:0)
    at com. opentok.android.Session$$ExternalSyntheticLambda30.run(Unknown Source:2)
    at android.os.Handler.handleCallback(Handler.java:883)
    at android.os.Handler.dispatchMessage (Handler.java: 100)
    at android.os.Looper.Loop(Looper. java: 214)
    at com.facebook.react.bridge.queue.MessageQueueThreadImpL$4.run(MessageQueue
```

**Relevant logs and/or screenshots**
>  
![Screenshot 2023-01-09 at 3 27 27 PM](https://user-images.githubusercontent.com/77872909/211298039-1191ab6b-7f6b-4a14-8896-b7b60f2f3d8d.png)

**Comment by abdulajet** [comment]
can you try the lastest version of the library please?

**Comment by KevinMac19** [comment]
> can you try the lastest version of the library please?

Yes I have tried with latest version also. But same issue is happening.

**Comment by pietgk** [comment]
any status update regarding this issue?

**Comment by beHaze** [comment]
We have released 2.27.6, try with this version. Please reach out to Vonage support (https://api.support.vonage.com/hc/en-us) if you’re still facing issues.

**Comment by beHaze** [comment]
We haven't heard back from you for a while, so we're closing this issue for now. Please, reach out if you still need help.
